dev/test-studio/package.json

```diff
@@ -14,6 +14,7 @@
   "dependencies": {
     "@portabletext/react": "^1.0.6",
     "@sanity/icons": "1.3.7-next.18",
+    "@sanity/color": "^2.1.16",
```

Comment:
This version should be the same as other `@sanity/color` deps: `"@sanity/color": "2.1.17-next.18"`